package.json

```diff
@@ -38,23 +38,22 @@
     "c8": "^8.0.1",
     "clean-publish": "^4.2.0",
     "cssnano": "^6.0.1",
+    "jiti": "^1.21.0",
     "jsdoc-to-markdown": "^8.0.0",
     "postcss": "^8.4.31",
     "postcss-import": "^15.1.0",
     "postcss-nested": "^6.0.1",
     "standard": "^17.1.0",
     "sugarss": "^4.0.1",
-    "ts-node": "^10.9.1",
-    "tsm": "^2.3.0",
     "typescript": "5.2.2",
     "uvu": "^0.5.6"
   },
   "peerDependencies": {
-    "postcss": ">=8.0.9",
-    "ts-node": ">=9.0.0"
+    "jiti": ">=1.21.0",
```

Comment:
There are also `tsm` and `tsx` tools to run TS.

Did you tried them to compare with `jiti`? (Just curious, maybe I will migrate from `tsm` to `jiti` too)

Reply:
I found https://github.com/privatenumber/ts-runtime-comparison

What do you think of using `tsx`?

Reply:
I don't think tsm/tsx provide any way to require specific files (or register/unregister loaders in node < 21).